api/apps/v1alpha1/common_types.go

```diff
@@ -47,9 +48,10 @@ type Metrics struct {
 
 // ServiceMonitor defines attributes to create a service monitor
 type ServiceMonitor struct {
-	AdditionalLabels map[string]string `json:"additionalLabels,omitempty"`
-	Interval         promv1.Duration   `json:"interval,omitempty"`
-	ScrapeTimeout    promv1.Duration   `json:"scrapeTimeout,omitempty"`
+	AdditionalLabels      map[string]string `json:"additionalLabels,omitempty"`
```

Comment:
AdditionalLabels are not labels appended to the ServiceMonitor metadata, but instead they are additional labels added to the metrics collected from the target.

Reply:
Noted